StarWars.Core/Services/Concrete/SwapiService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using StarWars.Core.Const;
+using StarWars.Core.Models;
+using StarWars.Core.Services.Abstract;
+
+namespace StarWars.Core.Services.Concrete;
+
+public sealed class SwapiService : ISwapiService
+{
+    private readonly HttpClient _httpClient = new();
+
+    public async Task<CharacterModel[]> GetStarWarsCharactersAsync()
+    {
+        var swapiResponse = await GetSwapiEndpointsAsync();
```

Comment:
Let's talk about namings

GetSwapiEndpointsAsync - in my opinion method with this name should return Task<string> with endpoint url.